test/integration/functional_test.go

```diff
@@ -816,8 +836,34 @@ func validateSSHCmd(ctx context.Context, t *testing.T, profile string) {
 	if NoneDriver() {
 		t.Skipf("skipping: ssh unsupported by none")
 	}
-	want := "hello\n"
-	rr, err := Run(t, exec.CommandContext(ctx, Target(), "-p", profile, "ssh", fmt.Sprintf("echo hello")))
+	want := "/home/docker\n"
+
+	rr, err := Run(t, exec.CommandContext(ctx, Target(), "-p", profile, "ssh", "pwd"))
+
+	if ctx.Err() == context.DeadlineExceeded {
+		t.Errorf("failed to run command by deadline. exceeded timeout : %s", rr.Command())
+	}
+
+	if err != nil {
+		t.Errorf("failed to run an ssh command. args %q : %v", rr.Command(), err)
+	}
+
+	if rr.Stdout.String() != want {
+		t.Errorf("expected minikube ssh command output to be -%q- but got *%q*. args %q", want, rr.Stdout.String(), rr.Command())
+	}
+
+	// testing hostname as well
```

Comment:
This comment is missing the "why" aspect. Why do we care that the hostname matches the profile exactly?